apps/web/src/server/views/applications/components/published-tag.component.njk

```diff
@@ -1,11 +1,6 @@
-{% macro publishedTag(publishedStatus) %}
-	{% set publishedTagColor = '' %}
-	{% set publishedLabel = publishedStatus %}
-
-	{% switch (publishedStatus) %}
```

Comment:
uh, yeah, you're very right! We'll need to come back on this template when we develop the Unpublishing so for now I added a //TODO to remember that this could need to be changed to expect many colours.